com.chartboost.mediation/Documentation/faq.md

```diff
@@ -0,0 +1,23 @@
+# Frecuently Asked Questions
+
+## Is the minimum iOS version the same for all ad adapters?
+
+No, each adapter might have a different minimum iOS version. Most of them, support iOS 10 as their minimum version. The biggest exception being Amazon Publisher Services, which requires minimum iOS 12. 
+
+Depending on your adapters, you might need to modify this value accordingly. For information on how to set this values for Unity, visit the [Unity Documentation](https://docs.unity3d.com/2017.3/Documentation/Manual/class-PlayerSettingsiOS.html).
+
+## Encountering duplicate Android libraries
+
+Chartboost Mediation provides multiple partner integrations with ease, we make sure that every possible integration is as easy as it can be. As such, we try to avoid integrations that might cause duplicate libraries for Android, since not Unity developers might be familiar on how to resolve this issue. 
```

Comment:
```suggestion
Chartboost Mediation provides multiple partner integrations with ease, we make sure that every possible integration is as easy as it can be. As such, we try to avoid integrations that might cause duplicate libraries for Android, since not many Unity developers might be familiar on how to resolve this issue. 
```